### PR TITLE
Add support for SMART app cards

### PR DESCRIPTION
**Pull request by nschwertner**
For the time being this works for apps based on the SMART JS client launched against the CDS Hooks demo harness' open FHIR API server

### CODE REVIEW

File: webpack.dev.config.js [hunk 1]
```diff
@@ -10,34 +11,34 @@ module.exports = {
...
   },
   output: {
-    path: "./build",
+    path: path.join(__dirname, "build"),
     filename: "bundle.js"
   },
   plugins: [
     new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoErrorsPlugin(),
+    new webpack.NoEmitOnErrorsPlugin(),
```
Comment (zplata) [inline webpack.dev.config.js:19]: This change is related to changing from webpack 1 to webpack 2. Can we make that version bump in the `package.json` accordingly (webpack ^2.6.1)? That way, anyone else cloning and `npm install`ing this project will be able to run `npm run dev` without an error. 

Can we also make that change in the prod webpack file as well?

Edit: Probably should include the version bump for the webpack dev server as well ( webpack-dev-server ^2.5.0)

File: webpack.dev.config.js [hunk 2]
```diff
@@ -10,34 +11,34 @@ module.exports = {
...
     new webpack.DefinePlugin({
       "runtime.CDS_HOOKS_URL": JSON.stringify(process.env.CDS_HOOKS_URL || "http://localhost:8081"),
       "runtime.FHIR_URL": JSON.stringify(process.env.FHIR_URL || "https://sb-fhir-dstu2.smarthealthit.org/api/smartdstu2/open")
     })
   ],
   resolve: {
-    modulesDirectories: ['node_modules'],
+    modules: ['node_modules'],
```
Comment (zplata) [inline webpack.dev.config.js:26]: This change should also be in the webpack prod config file

**Comment by zplata** [comment]
Can you run `npm run build` to get the build/bundle.js file with these incorporated changes?

**Comment by zplata** [comment]
Awesome, thanks @nschwertner! Two things to note:
1) Can you run `npm run build` to get the new bundled build file and the two auto-generated svg files associated?
2) Can you create another PR that contains that new bundled build file and the two auto-generated SVG files associated into the `gh-pages` branch? This is temporary as we plan to use Travis CI to auto-deploy to `gh-pages` soon.

**Comment by zplata** [comment]
@nschwertner Sorry - one last thing. I just tried running your bundle locally and we might need to lock down on a `react-day-picker` dependency. The app is failing to run locally due to undefined referencing referencing this library. Can you uninstall `react-day-picker` and lock down to `"react-day-picker": "~1.2.0"` in the `package.json` and re-build?

**Comment by nschwertner** [comment]
Done

**Review by zplata — APPROVED** [review]

**Review by kpshek — APPROVED** [review]
Thank you @nschwertner!

**Pull request merged**